fluency-fluentd/src/main/java/org/komamitsu/fluency/fluentd/ingester/FluentdIngester.java

```diff
@@ -71,12 +66,15 @@ public void ingest(String tag, ByteBuffer dataBuffer)
         ByteBuffer headerBuffer = ByteBuffer.wrap(header.toByteArray());
 
         if (config.isAckResponseMode()) {
-            byte[] uuidBytes = UUID.randomUUID().toString().getBytes(CHARSET);
-            ByteBuffer optionBuffer = ByteBuffer.wrap(objectMapper.writeValueAsBytes(new RequestOption(dataLength, uuidBytes)));
+            // The spec says to encode a 128 bit value as base64, but fluent-bit currently
+            // uses a 32 char long hex string, so we do the same for now.
+            String token = UUID.randomUUID().toString().replace("-", "");
```

Comment:
ACK token formats in Fluentd and Fluent Bit are different and there is no validation of it. How about simply passing a UUID hex string as is without removing hyphen? If Fluentd and Fluent Bit implement a validation of ACK token in the future, it would be time to change this format in Fluency as well.

Reply:
Fixed. I still think it is better to comply with the spec straight away, but I am also fine with going ahead like this for now.

Reply:
Thanks!
Let's follow the spec later after Fluentd and Fluent bit use a consistent format.